src/pages/starter-kit/checkout/getting-started.md

```diff
@@ -27,6 +27,8 @@ You must install or have access to the following prerequisites to develop with t
 - Access to the [Adobe Developer Console](https://console.adobe.io/) with an App
   Builder license. If you do not have access to the Adobe Developer Console or App Builder, refer to [get access to App Builder](https://developer.adobe.com/app-builder/docs/overview/getting_access/#get-access-to-app-builder).
 
+- If you intend to use the Admin UI SDK, you must also complete the [Admin UI SDK installation process](../../admin-ui-sdk/installation.md).
```

Comment:
The Checkout Starter Kit recommends a minimum version of v3.0 of the Admin UI SDK for PaaS, as this is the version that supports authentication for actions (Fyi [CEXT-4496](https://jira.corp.adobe.com/browse/CEXT-4496)). Do you think we should clarify this somewhere?
Not sure if we can just swap the version sample in Admin UI SDK as it's applicable for all PaaS users, but leaving a thought:
https://developer.adobe.com/commerce/extensibility/admin-ui-sdk/installation/#adobe-commerce-on-cloud-infrastructure
```
composer require "magento/commerce-backend-sdk": ">=3.0"
```

Reply:
i've added the command in a note